components/ExerciseDetails/index.js

```diff
@@ -2,40 +2,108 @@ import styled from "styled-components";
 
 export default function ExerciseDetails({ exercise }) {
   return (
-    <>
-      <figure>
+    <Container>
+      <ImageWrapper>
         <ExerciseImage src={exercise.imageUrl} />
+      </ImageWrapper>
+      <TitleAndMuscleWrapper>
+        <ExerciseTitle>{exercise.name}</ExerciseTitle>
         <ExerciseDetailsMuscle>
-          {exercise.muscleGroups.join(" ")}
+          {exercise.muscleGroups.map((muscle, index) => (
+            <MuscleGroup key={index}>{muscle}</MuscleGroup>
```

Comment:
It's not a good practice to use the index as the key. This can have negative effect on the performance of react. Best is using an id that comes from the object. In your case you can use the muscle itself because it will always be unique.